### ISSUE TITLE
react-flow cannot be installed in a fresh create-react-app app

### ISSUE THREAD

**Issue by Ark-kun** [wbkd/react-flow#2047]
### Describe the Bug

```
npm install react-flow-renderer --save
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: pipeline-editor-vscode@0.1.0
npm ERR! Found: react@18.0.0
npm ERR! node_modules/react
npm ERR!   react@"^18.0.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"16 || 17" from react-flow-renderer@10.0.8
npm ERR! node_modules/react-flow-renderer
npm ERR!   react-flow-renderer@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

### Steps to Reproduce the Bug or Issue

`npx create-react-app my-app`
`npm install react-flow-renderer`

### Expected behavior

I expect react-flow to be installable

### Platform

- OS: Windows

**Comment by chrtze** [comment]
We have not yet updated to React v18. Until we do, you would need to install React Flow with `--legacy-peer-deps`.

**Comment by FateRiddle** [comment]
@chrtze When is it expected to be happening? Popular boilerplates are defaulting to react 18 version, nextjs 12 even demand it with dependency range.

**Comment by moklick** [comment]
@FateRiddle please see #2067

**Comment by MLNW** [comment]
> @FateRiddle please see #2067

The linked MR has been merged just a few hours ago

**Comment by moklick** [comment]
Fixed with v10.2.0
